cadence/contracts/LendingComptroller.cdc

```diff
@@ -173,7 +171,7 @@ pub contract LendingComptroller {
         /// @Param supplyUnderlyingAmountScaled - Supply amount scaled by 1e18
         /// @Return error code
         ///
-        pub fun supplyAllowed(
+        access(all) fun supplyAllowed(
```

Comment:
`view`?

Reply:
Cannot be made view as non-view operation is performed in `callerAllowed()`, which checks if the caller is from a designated lending pool contract.